h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var Controller = require('../base/controller');
+var LozengeHelper = require('../util/lozenge-helper');
+
+class LozengeController extends Controller {
+  constructor(el) {
+    super(el);
+    el.querySelector('.js-lozenge-close').addEventListener('mousedown', () => {
+      // remove the DOM element
+      el.remove();
+      // update input field
+      this._inputHidden = document.querySelector('.js-search-bar').querySelector('.js-search-bar__input-hidden');
```

Comment:
This couples the lozenge controller to the structure of the document. What I would suggest doing instead is passing in callbacks (eg. An `onClick` function) via the `options` argument to the constructor (controllers now take two arguments, see `controller.js`)